common/common-rest/src/main/java/io/servicecomb/common/rest/codec/param/BodyProcessorCreator.java

```diff
@@ -63,7 +63,7 @@ public Object getValue(HttpServletRequest request) throws Exception {
       }
 
       String contentType = request.getContentType();
-      if (contentType != null && !contentType.startsWith(MediaType.APPLICATION_JSON)) {
+      if (contentType != null && !contentType.toLowerCase().startsWith(MediaType.APPLICATION_JSON)) {
```

Comment:
1.toLowerCase(Locale.US)?
2.where is accept bug fix?

Reply:
I prefer to use Locale.ROOT(recommended by java 8) than Locale.US or Locale.ENGLISH ( by java 7 and below ) ,but this two would be alright.